common/build/eslint-config-fluid/src/test/enforce-no-member-release-tags/enforce-no-member-release-tags.test.js

```diff
@@ -0,0 +1,294 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const assert = require("assert");
+const path = require("path");
+const { ESLint } = require("eslint");
+
+describe("ESLint No Release Tag Rule Tests", function () {
+	function createESLintInstance(config) {
+		return new ESLint({
+			useEslintrc: false,
+			overrideConfig: config,
+			rulePaths: [path.join(__dirname, "../../custom-rules")],
+		});
+	}
+
+	it("Should report errors for including release tags inside the class declaration", async function () {
+		const eslint = createESLintInstance({
+			rules: {
+				"no-member-release-tags": ["error"],
+			},
+			parser: "@typescript-eslint/parser",
+			parserOptions: {
+				project: path.join(__dirname, "../tsconfig.json"),
+			},
+		});
```

Comment:
That's a great point. Since this test suite covers the `no-member-release-tags` rule, I think I can move the configuration object inside the `createESLintInstance` function and also remove the `config` parameter.